ocp_install_env.sh

```diff
@@ -130,6 +130,22 @@ EOF
   fi
 }
 
+function apiVIP_ingressVIP_dualstack() {
+  # dual stack will be added 4.12, let's keep compat mode
+  # The char , is used for multiple vips which is supported only in dualstack mode
+  if [[ "${API_VIP}" == *","* ]]; then
+cat <<EOF
+    apiVIPs: ${API_VIP}
+    ingressVIPs: ${INGRESS_VIP}
+EOF
+  else
+cat <<EOF
+    apiVIP: ${API_VIP}
+    ingressVIP: ${INGRESS_VIP}
```

Comment:
When https://github.com/openshift/installer/pull/5798 got merged, you can also specify the single vip in the plural field. (-> no need for `apiVIP` and `ingressVIP` anymore). E.g.:
```
    apiVIPs:
    - apiVIP
    ingressVIPs:
    - ingressVIP
```
This allows you to have the same rendering for one and multiple vips, probably too.

Reply:
We still need support for older releases in dev-scripts so we need to have a way to use the old field names.

However, I'd prefer to look at the openshift release number to determine which we need. Even in the case of a single VIP in 4.12, we should still be using the new fields (the current implementation would always use the old fields for single VIP deployments, IIUC). At some point the deprecated ones will go away and we'd have to revisit this anyway. Better to do it now when it's fresh in our minds.